src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -230,6 +230,7 @@ void CoilCoolingDXCurveFitPerformance::simulate(EnergyPlus::EnergyPlusData &stat
     Real64 reportingConstant = state.dataHVACGlobal->TimeStepSys * Constant::rSecsInHour;
     this->recoveredEnergyRate = 0.0;
     this->NormalSHR = 0.0;
+    this->oneTimeAvailSchedSetup(state);
```

Comment:
This new function propagates the availability schedule specified in the parent DX coil to each operating mode once.